src/common/src/util/epoch.rs

```diff
@@ -18,10 +18,12 @@ use std::time::{Duration, SystemTime};
 
 use parse_display::Display;
 
+static UNIX_SINGULARITY_DATE_SEC: u64 = 1_617_235_200;
+
 /// `UNIX_SINGULARITY_DATE_EPOCH` represents the singularity date of the UNIX epoch:
 /// 2021-04-01T00:00:00Z.
 pub static UNIX_SINGULARITY_DATE_EPOCH: LazyLock<SystemTime> =
-    LazyLock::new(|| SystemTime::UNIX_EPOCH + Duration::from_secs(1_617_235_200));
+    LazyLock::new(|| SystemTime::UNIX_EPOCH + Duration::from_secs(UNIX_SINGULARITY_DATE_SEC));
```

Comment:
Do we need LazyLock? Seems both operaands are consts? https://doc.rust-lang.org/stable/std/time/struct.Duration.html#method.from_secs

Reply:
> Do we need LazyLock? Seems both operaands are consts? https://doc.rust-lang.org/stable/std/time/struct.Duration.html#method.from_secs

guess this is because of `SystemTime`

Reply:
Yes, the operation `SystemTime` + `Duration` is unfortunately not const. In particular, const fns in traits such as `Add` still do not exist https://github.com/rust-lang/rfcs/pull/2632